### ISSUE TITLE
Solution not working on Win11, HP Victus

### ISSUE THREAD

**Issue by Staals** [jingyu9575/remap-omen-key#16]
Dear creator,

Thanks for your efforts! I would really like to replace the Omen key on my HP Victus 16-e0xx with the home key. However, the two offered solutions both do not seem to work. 
When I try the standard approach, the registry key I'm supposed to edit does not exist. I searched the registry but haven't been able to find anything like it. I think it might be because the HP program launched by the Omen key changed to a Microsoft Store App? It's called the Omen Gaming Hub now.
When I try the alternative approach, the .exe starts like normal after importing the certificates, but the home key is not remapped. The executable does not show up in the task manager, but when I try to remove the custom client executable it's reported as still running.
If there's any workarounds available they would be very very welcome! 
Thanks in advance,

Staals

**Comment by dx07** [comment]
@Staals also concerned by this issue! @jingyu9575 if you could help us with it 🙏🙏

**Comment by RinceMG** [comment]
Yep, the same issue with HP Victus 16-e1xxx. It seems like the Omen key is not F24 anymore. I can't get the key code or remap it.
Any solutions?

**Comment by catinwarmhands** [comment]
Same problem here
